core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -567,15 +567,6 @@ sealed abstract class Chunk[+A] extends ChunkLike[A] with Serializable { self =>
     (s, builder.result())
   }
 
-  /**
-   * Statefully and effectfully maps over the elements of this chunk to produce
-   * new elements.
-   */
-  final def mapAccumM[R, E, S1, B](s1: S1)(f1: (S1, A) => ZIO[R, E, (S1, B)])(implicit
```

Comment:
I remove this method in this PR as there's already a `mapAccumZIO` method, but I'm not 100% sure whether it's the right thing to do.

Reply:
That's the right thing to do.